scripts/jenkins/Dockerfile

```diff
@@ -35,21 +35,22 @@ RUN export OPENMPI_VERSION=4.0.0 && \
 
 # Install Kokkos
 ENV KOKKOS_DIR=/opt/kokkos
-RUN export KOKKOS_VERSION=2.9.00 && \
-    export KOKKOS_URL=https://github.com/kokkos/kokkos/archive/${KOKKOS_VERSION}.tar.gz && \
-    export KOKKOS_ARCHIVE=kokkos-${KOKKOS_VERSION}.tar.gz && \
-    export KOKKOS_SOURCE_DIR=kokkos-${KOKKOS_VERSION} && \
+RUN KOKKOS_VERSION=master && \
```

Comment:
should we fix to 3.0?

Reply:
There is no master tag ;-)

Reply:
Neither will there is a 3.0 one.

Reply:
You can use the revision umber if you want to be explicit and replace with a tag later.